Consistency of examples (#200)
arndt-s/ietf-identity-chaining: draft-ietf-oauth-identity-chaining.md

### Token Exchange Response

All of {{Section 2.2 of RFC8693}} applies. In addition, the following applies to implementations that conform to this specification:

* The "aud" claim in the returned JWT authorization grant MUST identify the requested authorization server in trust domain B. This corresponds with Point 3 in {{Section 3 of RFC7523}} and is there to reduce misuse and to prevent clients from, among other things, presenting access tokens as an authorization grant to an authorization server in trust domain B.

* The "aud" claim included in the returned JWT authorization grant can identify multiple authorization servers, provided that trust relationships exist with them. However, it is RECOMMENDED that the "aud" claim is restricted to a single authorization server in trust domain B to prevent an authorization server from presenting the client's authorization grant to an authorization server in a different trust domain. For example, this will prevent the authorization server in trust domain B from presenting the authorization grant it received from the client in trust domain A to the authorization server for trust domain C.

### Example

> The example below shows a message invoked by a client in trust domain A to perform token exchange with the authorization server in trust domain A (https://as.a.example/auth) to receive a JWT authorization grant for an authorization server in trust domain B (https://as.b.example/auth). Extra line breaks and indentation in the examples are for display purposes only. The JWT signatures are for illustration only and are not verifiable

Author: ```suggestion
The example below shows a message invoked by a client in trust domain A to perform token exchange with the authorization server in trust domain A (https://as.a.example/auth) to receive a JWT authorization grant for an authorization server in trust domain B (https://as.b.example/auth). Extra line breaks and indentation in the examples are for display purposes only. The JWT signatures are for illustration only and are not verifiable.
```

> The example below shows a message invoked by a client in trust domain A to perform token exchange with the authorization server in trust domain A (https://as.a.example/auth) to receive a JWT authorization grant for an authorization server in trust domain B (https://as.b.example/auth). Extra line breaks and indentation in the examples are for display purposes only. The JWT signatures are for illustration only and are not verifiable.

Author: ```suggestion
The example below shows a message invoked by a client in trust domain A to perform token exchange with the authorization server in trust domain A (https://as.a.example/auth) to receive a JWT authorization grant for an authorization server in trust domain B (https://as.b.example/auth). Extra line breaks and indentation in the examples are for display purposes only. The JWT signature is for illustration only and is not verifiable.
```

### Example

> The examples below show how a client in trust domain A presents an authorization grant to the authorization server in trust domain B (https://as.b.example/auth) to receive an access token for a protected resource in trust domain B. Extra line breaks and indentation in the examples are for display purposes only. The JWT signatures are for illustration only and are not verifiable

Author: ```suggestion
The examples below show how a client in trust domain A presents an authorization grant to the authorization server in trust domain B (https://as.b.example/auth) to receive an access token for a protected resource in trust domain B. Extra line breaks and indentation in the examples are for display purposes only. The JWT signatures are for illustration only and are not verifiable.
```

> The examples below show how a client in trust domain A presents an authorization grant to the authorization server in trust domain B (https://as.b.example/auth) to receive an access token for a protected resource in trust domain B. Extra line breaks and indentation in the examples are for display purposes only. The JWT signatures are for illustration only and are not verifiable.

Author: ```suggestion
The examples below show how a client in trust domain A presents an authorization grant to the authorization server in trust domain B (https://as.b.example/auth) to receive an access token for a protected resource in trust domain B. Extra line breaks and indentation in the examples are for display purposes only. The JWT signature is for illustration only and is not verifiable.
```

Cache-Control: no-cache, no-store

{
  "access_token":"eyJhbGciOiJFUzI1NiIsInR5cCI6ImF0K2p3dCJ9.eyJpc3MiOiJod
  HRwczovL2FzLmIuZXhhbXBsZS9hdXRoIiwiZXhwIjoxNjk1Mjg3NzUyLCJpYXQiOjE2OTU
  yODc2OTIsInN1YiI6ImRvZS5qb2huQGIuZXhhbXBsZSIsImF1ZCI6Imh0dHBzOi8vYi5le
  GFtcGxlL2FwaSJ9.Fd0SMTM_Ylv9RM-Y65g4Fn5X6Lk8A8t8ndZ39zaEykvJpLp4-cwcsb

>   4018gZBHHG8AI5IDcu_hoLs0jdhwrJtA",

Reviewer: let's just make this one not a JWT to further distance ourself from possible mistakes and/or prospective confusion
 
```suggestion
  "access_token":"b./77-.ehs96cFAx~ac00~qG+z8N_x/4.RpL-2Ykc4~",
```
